Formula/azure-cli-2.rb

```diff
@@ -0,0 +1,75 @@
+require "tmpdir"
+
+class AzureCli2 < Formula
+  include Language::Python::Virtualenv
+
+  desc "Azure CLI 2.0"
+  homepage "https://github.com/Azure/azure-cli"
+  url "https://azurecliprod.blob.core.windows.net/releases/azure-cli_packaged_0.1.5.tar.gz"
+  sha256 "cb65651530544c43343e2c80b0d96a681ad0a07367e288ef9fae261093c497f3"
+  depends_on "python"
+
+  # Apply the 'az component' patch
+  patch do
+    url "https://azurecliprod.blob.core.windows.net/patches/patch_0.1.5_component_custom.diff"
+    sha256 "d61ef29ace9bbdfef9a25dfbb1f475225bbca174263c8f863ee70f87d0a78bbe"
+  end
+
+  # Apply pip module search patch
+  patch do
+    url "https://azurecliprod.blob.core.windows.net/patches/patch_0.1.5_pkg_util.diff"
+    sha256 "ea0879280dbb3074d464752c27bfe01a7673da14137a4c4315d1938a0d05a03e"
+  end
+
+  def completion_script; <<-EOS.undent
```

Comment:
Thanks for the suggestion.
Tracked in upstream with https://github.com/Azure/azure-cli/issues/1768.
This will be changed in the next release then this can be removed.